src/rabbitmq.js

```diff
@@ -671,7 +665,32 @@ class RabbitMQ {
         this.subscribed = this.subscribed.add(subscribedKey)
       })
   }
-
+  /**
+   * Create payload object with `jobBuffer` and `jobMeta` props
+   * @private
+   * @param {Object} content Content to send.
+   * @return {Object} payload with `jobBuffer` and `jobMeta` props
+   */
+  static buildPayload (content: Object) {
+    const jobMeta = {
+      appId: this.name,
+      timestamp: Date.now(),
+      headers: {}
+    }
+    const ns = getNamespace('ponos')
+    jobMeta.headers.previousEvent = ns && ns.get('previousEvent')
```

Comment:
also here you can convert `currentWorkerName` to `sendingWorkerName` or `publisherWorkerName` or `publisherName`